R/mappings.R

```diff
@@ -19,8 +19,8 @@
 #' @export
 identical_aes <- function(a1, a2) {
   # strip environments associated with the aesthetics before comparing
-  a1 <- lapply(a1, rlang::as_name)
-  a2 <- lapply(a2, rlang::as_name)
+  a1 <- lapply(a1, rlang::as_label)
+  a2 <- lapply(a2, rlang::as_label)
```

Comment:
```suggestion
  a1 <- lapply(a1, deparse)
  a2 <- lapply(a2, deparse)
```

Reply:
Btw, this where `waldo::compare(aes(x = rnorm(1))$x, aes(x = rnorm(2))$x)` lands, which partly motivated choosing `deparse()`.

https://github.com/r-lib/waldo/blob/8e22b227d92eed12a178fc4a3507aec319cd6279/R/compare.R#L269-L273